ros2_control_demo_robot/description/rrbot_system_position_only.urdf.xacro

```diff
@@ -5,7 +5,8 @@ Copied and modified from ROS1 example:
 https://github.com/ros-simulation/gazebo_ros_demos/blob/kinetic-devel/rrbot_description/urdf/rrbot.xacro
 -->
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="2dof_robot">
-
+  <xacro:arg name="simu" default="false" />
```

Comment:
nitpick, but `simu` sounds kinda weird. I'd propose either just `sim` or something like `use_sim`.

Reply:
Thanks. Fixed in 5e726d4f7000c2162c1e647d43d20339fee46f58